Add article about changes in coin selection module (#114)
bitcoindevkit/bitcoindevkit.org: docs/_blog/improving_coin_selection_in_BDK.md

---
title: "Improving coin selection in BDK"
description: "A brief description of the work done in the coin selection module in BDK during Summer of Bitcoin 2022"
date: "2022-08-17"
tags: ["coin_selection", "BDK", "development", "summer_of_bitcoin"]
authors:
  - "César Alvarez Vallero"
hidden: true
draft: false
---

As a project designed to use as a build tool in wallet development, one of the
main things that BDK provides is the coin selection module. The purpose of the
module is to select the group of utxos to use as inputs for the transaction.
When you coin select you must consider cost, size and traceability.

- What are those costs?

  Principally fees. Determined by the satisfaction size required by each of the
  inputs. But are also related to the change outputs generated. Change outputs

>   are not part of the inputs, but they must be considered during coin selection

Reviewer: ```suggestion
  is not part of the inputs, but must be considered during coin selection
```
